### PR TITLE
Issue#366 updates IN1 class, subscriberId, order

### PR DESCRIPTION
**Pull request by cragun47**
Signed-off-by: Brian Cragun <cragun@us.ibm.com>

Updates for IN1 to class, subscriberId, and order.

(Also fixed some typos in HL7ConditionFHIRConversionTest)

### CODE REVIEW

File: src/main/resources/hl7/secondary/Class.yml [hunk 1]
```diff
@@ -3,25 +3,26 @@
...
+   # Constants because Class is currently only used by Coverage 
+   constants: 
+      system: 'http://terminology.hl7.org/CodeSystem/coverage-class'
+      code: 'group'
+      display: 'Group'
 
 name:
      type: STRING
-     valueOf: $className
+     valueOf: XON.1 | $className
```
Comment (LisaWellman) [inline src/main/resources/hl7/secondary/Class.yml:26]: className is no longer needed

File: src/test/java/io/github/linuxforhealth/hl7/segments/Hl7FinancialInsuranceTest.java [hunk 2]
```diff
@@ -77,14 +79,19 @@ void testBasicInsuranceCoverageFields() throws IOException {
                 //    IN1.7.14 to Organization Contact Name .period.end
                 //    IN1.7.18 to Organization Contact telecom .rank 
                 + "|^^^^^333^4444444^^^^^^20201231145045^20211231145045^^^^1"
-                // IN1.8 to Coverage.class.value
-                // IN1.9.1 to Coverage.class.name
+                // IN1.8 to list element Coverage.class.value
+                // IN1.9.1 (1) to list element Coverage.class.name
+                // IN1.9.10 (1) to list element Coverage.class.value
+                // IN1.9.1 (2) to list element Coverage.class.name and value (because IN1.9.10 (2) does not exist)
```
Comment (LisaWellman) [inline src/test/java/io/github/linuxforhealth/hl7/segments/Hl7FinancialInsuranceTest.java:85]: Nice!

**Review by LisaWellman — APPROVED** [review]
Great tests.  
Just one cleanup item in yml

File: src/main/resources/hl7/secondary/Class.yml (hunk 1 above)
Comment (cragun47) [reply src/main/resources/hl7/secondary/Class.yml]: Fixed.

**Pull request merged**